docs/enhancements/infrastructure-monitoring-service-api/lifecycle.md

```diff
@@ -0,0 +1,530 @@
+	EntityType                   string                  `json:"entityType"`                   // M, 1, O-RAN entity type emitting the alarm:  This shall be unique per vendor ResourceType.model and ResourceType.version
+	Vendor                       string                  `json:"vendor"`                       // M, 1, Vendor of the Entity Type to whom this Alarm Dictionary applies. This should be the same value as in the ResourceType.vendor attribute.
+	ManagementInterfaceID        []ManagementInterfaceID `json:"managementInterfaceId"`        // M, 1..N, List of management interface over which alarms are transmitted for this Entity Type. RESTRICTION: For the O-Cloud IMS Services this value is limited to O2IMS.
+	PKNotificationField          []string                `json:"pkNotificationField"`          // M, 1..N, Identifies which field or list of fields in the alarm notification contains the primary key (PK) into the Alarm Dictionary for this interface; i.e. which field contains the Alarm Definition ID.
+	AlarmDefinition              []AlarmDefinition       `json:"alarmDefinition"`              // M, 1..N, List of alarms that can be detected against this ResourceType
+}
+```
+
+- AlarmDefinition
+```go
+// 3.2.6.2.9-1
+type AlarmDefinition struct {
+	AlarmDefinitionID     uuid.UUID               `json:"alarmDefinitionID"`               // M, 1, Provides a unique identifier of the alarm being raised.  This is the Primary Key into the Alarm Dictionary.
+	AlarmName             string                  `json:"alarmName"`                       // M, 1, Provides short name for the alarm.
+	AlarmLastChange       string                  `json:"alarmLastChange"`                 // M, 1, Indicates the Alarm Dictionary Version in which this alarm last changed.
+	AlarmChangeType       AlarmLastChangeType     `json:"alarmChangeType"`                 // M, 1, Indicates the type of change that occurred during the alarm last change; added, deleted, modified.
+	AlarmDescription      string                  `json:"alarmDescription"`                // M, 1, Provides a longer descriptive meaning of the alarm condition and a description of the consequences of the alarm condition. This is intended to be read by an operator to give an idea of what happened and a sense of the effects, consequences, and other impacted areas of the system
+	ProposedRepairActions string                  `json:"proposedRepairActions"`           // M, 1, Provides guidance for proposed repair actions.
+	ClearingType          ClearingType            `json:"clearingType"`                    // M, 1, Whether alarm is cleared automatically or manually
+	ManagementInterfaceID []ManagementInterfaceID `json:"managementInterfaceId,omitempty"` // M, 0..N, List of management interface over which alarms are transmitted for this Entity Type. RESTRICTION: For the O-Cloud IMS Services this value is limited to O2IMS.
```

Comment:
Slice? only O2IMS is applied to our extension.

Reply:
Just following the spec as spoke about it internally